### ISSUE TITLE
Anonymous session data not converted to user session on login: Emoji etc.

### ISSUE THREAD

**Issue by wbt** [govtrack/govtrack.us-web#142]
If I am not logged in and click on an emoji, it indicates that I've made this selection (even on returning to the bill page later) and increments the aggregate count that everyone can see, as expected.  However, if I then log in, the emoji selection data associated with the anonymous account does not get associated with my login.  Viewing the bill shows me that I have selected none, and clicking again registers the response and increases the count.  If I then log out, that starts another anonymous session.  By logging in and out, I can increment the emoji count as high as I want on any emoji on any bill.  

A possible (at least partial) fix for this would be to have the login script look for any stored data associated with an anonymous session, and convert that to the user's logged-in account.

**Comment by JoshData** [comment]
Yeah. That's true. The emojis aren't important enough to warrant the extra code needed to migrate the anonymous data on login/account creation, I don't think. I doubt it occurs (for users) very often.
